perl/lib/Lingy/nREPL.pm

```diff
@@ -56,6 +57,10 @@ sub new {
 sub op_eval {
     my ($self) = @_;
 
+    if (my $file = $self->{request}{file}) {
+        RT->env->set('*file*', $file);
```

Comment:
I considered, but hesitated because thought maybe it should by Lingy-fied on read rather than on write. But if it is supposed to be a Lingy string I'll go ahead and make this change.